test-tap/fixture/meta.js

```diff
@@ -1,9 +1,14 @@
 const {default: test, meta} = require('../..');
 
+test('meta is test.meta', t => {
+	t.is(meta, test.meta);
+});
```

Comment:
Is this test necessary? I added it to not have to add two tests for every property as was done for `file` (checking both `meta.property` and `test.meta.property`).

Reply:
I guess it won't hurt 😄 

I'm looking to remove this extra export in #2435.